misc/acrn-config/host/kconfig/pylint.conf

```diff
@@ -0,0 +1,570 @@
+[MASTER]
```

Comment:
A new file?

Reply:
It was pylint temporary file during scan, will delete it in source code.